src/utils/netdata-sdk/metrics-stream.ts

```diff
@@ -1,36 +1,43 @@
-import { Subject, from, empty } from "rxjs"
 import {
-  mergeMap, tap, catchError,
+  Subject, from, empty,
+} from "rxjs"
+
+import {
+  mergeMap, tap, catchError, startWith, switchMap,
 } from "rxjs/operators"
 
 import { axiosInstance } from "./axios-instance"
 
-const CONCURRENT_CALLS_LIMIT = 20
-
 interface FetchInputEvent {
   url: string
   params: { [key: string]: unknown }
```

Comment:
could be `type UnknownStringKeyT =  { [key: string]: unknown }`

Reply:
i like that idea. fixed